frontend/src/main/scala/bloop/data/WorkspaceSettings.scala

```diff
@@ -40,12 +36,27 @@ import com.github.plokhotnyuk.jsoniter_scala.macros.{JsonCodecMaker, CodecMakerC
  * we should skip the resolution of the plugin.
  */
 case class WorkspaceSettings(
-    semanticDBVersion: String,
-    supportedScalaVersions: List[String]
-)
+    semanticDBVersion: Option[String],
+    supportedScalaVersions: Option[List[String]],
+    refreshProjectsCommand: Option[List[String]]
```

Comment:
Nit: can we have a top-level field in the workspace settings json called `commands` and then define an object within it with the key `refreshProjects` and its value? I prefer this as it leaves the door open to new commands in the future.

Reply:
I've given this some thought, I retract what I said, there's no need to do this